pkg/config/setup/config.go

```diff
@@ -1916,13 +1918,14 @@ func ResolveSecrets(config pkgconfigmodel.Config, secretResolver secrets.Compone
 				return
 			}
 			if err := configAssignAtPath(config, settingPath, newValue); err != nil {
-				log.Errorf("could not assign to config: %s", err)
+				log.Errorf("Could not assign new value of secret %s to config: %s", handle, err)
```

Comment:
Printing the settingPath would be useful too.

Reply:
Added to the log !